src/main/webapp/app/modules/canvas/visualization/visualization-properties/partials/properties/visualization-title-properties.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { ReactText, useEffect, useState } from 'react';
+import { IRootState } from 'app/shared/reducers';
+import { connect } from 'react-redux';
+import { Form, Heading, Item, Picker, TextField, View, Well } from '@adobe/react-spectrum';
+import { getBorderList } from 'app/modules/canvas/visualization/visualization-modal/visualization-edit-modal/visualization-edit-modal-util';
+import { TitleProperties } from 'app/shared/model/visualMetadata.model';
+
+export interface IVisualizationTitlePropertiesProps extends StateProps, DispatchProps {
+  titleProperties: TitleProperties;
+}
+
+const VisualizationTitleProperties = (props: IVisualizationTitlePropertiesProps) => {
+  const borderList = getBorderList();
+  const [properties, setProperty] = useState([]);
```

Comment:
again set the default values using model

Reply:
default value is set in component